RELEASE.rst

```diff
@@ -63,6 +63,19 @@ and a `source distribution <https://packaging.python.org/guides/distributing-pac
 
      $ python3 -m twine upload dist/*
 
-#. Trigger a build of the docs on https://readthedocs.org/.
+#. Make sure the new version appears on https://motor.readthedocs.io/. If the
+   new version does not show up automatically, trigger a rebuild of "latest":
+   https://readthedocs.org/projects/motor/builds/
 
-#. Announce!
+#. Bump the version number to <next version>.dev0 in setup.py/__init__.py,
+   commit, push.
+
+#. Publish the release version in Jira.
+
+#. Announce the release on:
+    https://developer.mongodb.com/community/forums/c/community/release-notes/
```

Comment:
This line is still over indented. Also this link doesn't work for me, let's use: https://www.mongodb.com/community/forums/c/announcements/driver-releases

Reply:
Fixed